modules/virt-installing-virtio-drivers-installing-windows.adoc

```diff
@@ -17,15 +17,22 @@ installation method might differ between versions of Windows. See the
 documentation for the version of Windows that you are installing.
 ====
 
+.Prerequisites
+
+* A storage device containing the `virtio` drivers must be attached to the VM.
+
 .Procedure
 
-. Start the virtual machine and connect to a graphical console.
-. Begin the Windows installation process.
-. Select the *Advanced* installation.
-. The storage destination will not be recognized until the driver is loaded.
-Click `Load driver`.
-. The drivers are attached as a SATA CD drive. Click *OK* and browse the CD drive
- for the storage driver to load. The drivers are arranged hierarchically
-according to their driver type, operating system, and CPU architecture.
-. Repeat the previous two steps for all required drivers.
-. Complete the Windows installation.
+. In the Windows Guest OS, use the `File Explorer` to navigate to the `virtio-win` CD drive.
+. Double-click to run the appropriate installer for your VM:
+.. For a 64-bit vCPU, use the `virtio-win-gt-x64` installer. 32-bit vCPUs are no longer supported.
+. Optional: During the *Custom Setup* step, select the device drivers you want to install. The recommended driver set is selected by default.
```

Comment:
From context, I can't tell if if *Custom Setup* is a UI element or a literal `Custom Setup`. I wanted to point it out though. Perhaps clarifying if it is it a step or a menu item could help.